skiko/buildSrc/src/main/kotlin/tasks/configuration/NativeTasksConfiguration.kt

```diff
@@ -74,7 +74,7 @@ fun SkikoProjectContext.compileNativeBridgesTask(
                     Arch.Arm64 -> arrayOf(
                         "-target", if (isUikitSim) "arm64-apple-ios-simulator" else "arm64-apple-ios",
                         "-isysroot", if (isUikitSim) iphoneSimSdk else iphoneOsSdk,
-                        "-miphoneos-version-min=12.0"
```

Comment:
It seems `iphoneos-version-min` is outdated alias to `ios-version-min`. Should we align that?

Reply:
Could you please point me to the docs about ios-version-min? Where did you see that it's outdated?

Reply:
In fact, it's just alias. See https://clang.llvm.org/docs/ClangCommandLineReference.html:
> -mios-simulator-version-min=<arg>, -miphonesimulator-version-min=<arg>
-mios-version-min=<arg>, -miphoneos-version-min=<arg>
Set iOS deployment target

"iPhoneOS" is here for historical compatibility reasons.
See https://en.wikipedia.org/wiki/IOS_version_history
> iPhone OS was renamed iOS following the release of the iPad, starting with iOS 4

Reply:
thanks. fixed